### PR TITLE
WINUI-3237-Added UG document for Ribbon Compact sizing.

### PR DESCRIPTION
**Pull request by BhaskarSuresh22**
* Added UG document for Compact Sizing feature of Ribbon control.
* Added the Contents on what is compact sizing, its features, how to implement it and the affected areas when compact sizing is applied.
* Added image of Ribbon control when compact sizing is applied.

### CODE REVIEW

File: winui/Ribbon/RibbonCompactSizing.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Ribbon Compact Sizing in WinUI Ribbon control | Syncfusion
+description: Learn here all about Ribbon Compact Sizing feature of Syncfusion WinUI Ribbon(SfRibbon) control with custom support and more.
```
Comment (laksmipriyamurugesan) [inline winui/Ribbon/RibbonCompactSizing.md:4]: Learn here all about the Ribbon Compact Sizing feature of Syncfusion WinUI Ribbon (SfRibbon) control with custom support and more.

File: winui/Ribbon/RibbonCompactSizing.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+There are two types of spacing support for WinUI Ribbon control.
+1. [Standard](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-standard-sizing) - to accommodate both touch and pointer input.
+2. [Compact](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-compact-sizing) - designed primarily to accommodate pointer input.
+
+The Compact sizing feature in the Ribbon control allows you to display the ribbon in a smaller, more condensed form. This can be useful for scenarios where you need to save space on the user interface or make the ribbon more accessible on smaller screens.
+
+## Applying compact sizing for Ribbon control
+
+Compact sizing is implemented through a unique resource dictionary that can be specified in your application at either the page level or a specific layout. The resource dictionary is available in the WinUI Nuget package.
```
Comment (laksmipriyamurugesan) [inline winui/Ribbon/RibbonCompactSizing.md:20]: Compact sizing is implemented by a unique resource dictionary that can be specified in your application at either the page level or a specific layout. The resource dictionary is available in the WinUI NuGet package.

File: winui/Ribbon/RibbonCompactSizing.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        <ResourceDictionary Source="ms-appx:///Syncfusion.Core.WinUI/Themes/DensityStyles/Compact.xaml" />
+    </Grid.Resources>
+</Grid>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Compact Sizing applied areas in Ribbon control
+
+When Compact styling is applied, elements such as buttons and other controls may be displayed at a smaller size, and layout elements such as height, width and spacing  may be rearranged to take up less space.
```
Comment (laksmipriyamurugesan) [inline winui/Ribbon/RibbonCompactSizing.md:54]: When Compact styling is applied, elements such as buttons and other controls can be displayed at a smaller size, and layout elements such as height, width, and spacing can be rearranged to take up less space.

File: winui/Ribbon/RibbonCompactSizing.md (hunk 1 above)
Comment (BhaskarSuresh22) [reply winui/Ribbon/RibbonCompactSizing.md]: Done the changes

File: winui/Ribbon/RibbonCompactSizing.md (hunk 2 above)
Comment (BhaskarSuresh22) [reply winui/Ribbon/RibbonCompactSizing.md]: Done the changes.

File: winui/Ribbon/RibbonCompactSizing.md (hunk 3 above)
Comment (BhaskarSuresh22) [reply winui/Ribbon/RibbonCompactSizing.md]: Done the changes.

**Review by jeganraj-m — APPROVED** [review]
changes are fine

**Pull request merged**